test/units/galaxy/test_collection.py

```diff
@@ -0,0 +1,688 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+import os
+import pytest
+import re
+import tarfile
+import tempfile
+import time
+import uuid
+
+from io import BytesIO, StringIO
+from units.compat.mock import MagicMock
+
+import ansible.module_utils.six.moves.urllib.error as urllib_error
```

Comment:
`ansible.module_utils.six` is a package, not sure you can import from moves like this.

Reply:
It seems to work but I've changed it in the latest commit based on other changes.